### ISSUE TITLE
Render crash when joining server

### ISSUE THREAD

**Issue by pwncc** [Johni0702/BetterPortals#130]
Render crash when  joining server
[crash-2019-07-16_16.16.10-client.txt](https://github.com/Johni0702/BetterPortals/files/3397513/crash-2019-07-16_16.16.10-client.txt)

**Comment by Johni0702** [comment]
Could you please give BP 0.2 a try and see if it still crashes?
If so, removing third-party mods as much as possible would be really helpful to allow me to reproduce it without 30 minutes of downloading random, mostly unrelated third-party mods.

**Comment by pwncc** [comment]
Hello, we will add 0.2 to the server when we can.

**Comment by bobodoustaud** [comment]
have the same issue but i have less mods

does you need the crash report?

**Comment by Johni0702** [comment]
@bobodoustaud yes, I need an updated crash report with latest BP (currently 0.2.3) and as few third-party mods as possible (so either none or just the conflicting mod).

**Comment by bobodoustaud** [comment]
ok here is a pastebin : https://pastebin.com/kyaMnpyQ

and the mod list is in the crash report

**Comment by Johni0702** [comment]
That's still BP 0.1.6 and still far too many mods.

**Comment by bobodoustaud** [comment]
ok with the new version, my screen is totally brown or black

first person
![2019-07-28_16 04 11](https://user-images.githubusercontent.com/49878442/62007748-9a421480-b151-11e9-94af-d76257617fbc.png)

third person
![2019-07-28_16 04 24](https://user-images.githubusercontent.com/49878442/62007753-b2199880-b151-11e9-9ca1-1abeb138a50d.png)

**Comment by Johni0702** [comment]
To quote myself:
> as few third-party mods as possible (so either none or just the conflicting mod)

If I were you, I'd start with your performance mods.

**Comment by bobodoustaud** [comment]
no changes without my optimisation mods (better fps, optifine ant foamfix) i will try in solo

**Comment by bobodoustaud** [comment]
with only forgelin and betterportal :

![2019-07-28_16 40 21](https://user-images.githubusercontent.com/49878442/62008156-7a611f80-b156-11e9-895c-83b2d01c0583.png)

no changes

**Comment by bobodoustaud** [comment]
the background colour changes when i hold an item:
![2019-07-28_16 42 23](https://user-images.githubusercontent.com/49878442/62008304-cdd36d80-b156-11e9-81c1-8d872e9e1097.png)

and a blue colour in the water:
![2019-07-28_16 45 46](https://user-images.githubusercontent.com/49878442/62008654-3cb0c680-b157-11e9-9a94-4bdd859f3adb.png)

when i go in the lava, its the same color as when i hold an item

**Comment by Johni0702** [comment]
Hm... strange... I've got no idea what's causing that.
Could you post your logs? Maybe there's something in there which explains it.

**Comment by bobodoustaud** [comment]
ok https://pastebin.com/bgnQY1tQ

**Comment by Johni0702** [comment]
I don't think you've got the right log. It's got 7 mods installed and contains a crash whereas your screenshot only has 6 and has evidently not crashed.

**Comment by bobodoustaud** [comment]
[2019-07-28-7.log.gz](https://github.com/Johni0702/BetterPortals/files/3439363/2019-07-28-7.log.gz)
this one?

**Comment by Johni0702** [comment]
Yeah, that looks like the correct one.
Unfortunately I've got no idea why that's happening and cannot reproduce it either.

**Comment by bobodoustaud** [comment]
i'm going to try on my another computer to see if it's work

**Comment by lag42** [comment]
The exact same thing is happening for me with just BP and Optifine
log [here](https://gist.github.com/lag42/44700448e0a701342b3597a4ec63a0f5)
